components/reddit/sources/new-comments-on-a-subreddit/new-comments-on-a-subreddit.js

```diff
@@ -0,0 +1,89 @@
+const common = require("../common");
+const { reddit } = common.props;
+
+module.exports = {
+  ...common,
+  key: "new-comments-on-a-subreddit",
+  name: "New comments on a subreddit",
+  description:
+    "Emits an event each time a new comment is added to a subreddit.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    ...common.props,
+    subreddit: {
+      propDefinition: [common.props.reddit, "subreddit"],
```

Comment:
You extract `reddit` from `common.props` from line 2 - can you use `reddit` directly here?

Reply:
THere was an instance where accesing `reddit` was problematic, and thought using `common.props.reddit` was the safe bet